js/main.js

```diff
@@ -0,0 +1,98 @@
+// Module imports
+import fetchCityData from "./current_weather.js";
+import getGeolocationWeather from "./geolocation.js";
+import { getCurrentWeatherString } from "./current_weather.js";
+
+// Retrieves user input
+// Passes input to fetchCityData function
+
+const searchBtn = document.querySelector("#submit");
+const input = document.querySelector("#search-query");
+
+searchBtn.addEventListener("click", () => {
+  const input = document.querySelector("#search-query").value;
+  fetchCityData(input);
+
+  document.querySelector("#search-query").value = "";
+});
+
+input.addEventListener("keypress", (event) => {
+  if (event.key === "Enter") {
+    event.preventDefault();
+    document.querySelector("#submit").click();
+  }
+});
+
+
+// Toggles celsius/fahrenheit
+
+const toggleBtn = document.querySelector("#toggle-temp");
+
+toggleBtn.addEventListener('click', () => {
+  if (document.querySelector("#toggle-temp").textContent === "Fahrenheit") {
+    document.querySelector("#toggle-temp").textContent = "Celsius";
+  } else {
```

Comment:
could be simplified to
document.querySelector('#toggle-temp').textContent = document.querySelector('#toggle-temp').textContent === 'Fahrenheit' ? 'Celsius' : 'Fahrenheit';